software/externals/cmake/externals.cmake

```diff
@@ -291,6 +291,12 @@ if(BUILD_PRIVATE_EXTERNALS)
     flycapture
     )
 
+  if(APPLE)
```

Comment:
sorry for all the requests... can you actually not include this?  I'm happy to have the code downloaded on apple, I'll just set the option not to build it.  It's nice to have downloaded so i can read it.